kie-server-parent/kie-server-api/src/test/java/org/kie/server/api/util/TaskQueryFilterSpecBuilderTest.java

```diff
@@ -0,0 +1,16 @@
+package org.kie.server.api.util;
+
+import org.junit.Test;
+
+public class TaskQueryFilterSpecBuilderTest {
```

Comment:
@domhanak I've implemented the test class and also added a unit-test for ProcessInstanceQueryFilterSpecBuilder: https://github.com/DuncanDoyle/droolsjbpm-integration/commit/870dcbf74c1523b130dec00e06ac7bf20aca71a7